src/shutdown.py

```diff
@@ -16,7 +16,10 @@
 
 
 def doCleanShutdown():
-    """Used to tell proof of work worker threads and the objectProcessorThread to exit."""
+    """
+    Used to tell proof of work worker threads and the
+    `.threads.objectProcessor` thread to exit.
```

Comment:
Well now it notifies all kinds of threads, so the reference to "worker" should be removed.